onelogin_rules/onelogin_brute_force_by_ip.yml

```diff
@@ -16,6 +16,13 @@ Threshold: 10
 DedupPeriodMinutes: 10
 Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
 Runbook: Analyze the IP they came from, and other actions taken before/after. Check if a user from this ip eventually authenticated successfully.
+SummaryAttributes:
+  - event_type_id
```

Comment:
The event_type_id will always be 6 here, is this a useful SummaryAttribute?

I believe that for several of these rules the `event_type_id` could only have one or two values, I think we may want  to drop it as a `SummaryAttribute` from any of those unless you had something specific in mind.

Reply:
Thought I got all of these but you're correct. Will remove `event_type_id` where we are checking for a specific id